test/plugins/sensor_demo/custom_system/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+find_package(ignition-cmake2 REQUIRED)
+project(CustomSystem)
+
+ign_find_package(ignition-plugin1 REQUIRED COMPONENTS register)
+set(IGN_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
+
+find_package(ignition-sensors6 REQUIRED)
+set(IGN_SENSORS_VER ${ignition-sensors6_VERSION_MAJOR})
+
+find_package(ignition-rendering6 REQUIRED)
+set(IGN_RENDERING_VER ${ignition-rendering6_VERSION_MAJOR})
```

Comment:
I think these lines can be removed for the same reasons I mentioned about using `find_package`.

Reply:
Removed these lines, but CMake cannot resolve the name ``${IGN_GAZEBO_VER}`` here, hence I've hardcoded ``ignition-gazebo6-rendering`` in ``target_link_libraries``. How do I resolve this ?